prime-angular-frontend/src/app/shared/models/health-authority.model.ts

```diff
@@ -12,4 +13,5 @@ export interface HealthAuthority {
   privacyOffice: PrivacyOffice;
   technicalSupports: Contact[];
   pharmanetAdministrators: Contact[];
+  healthAuthorityOrganizationAgreementDocument: BaseDocument;
```

Comment:
Is this variable name too long?  Is `healthAuthorityOrgAgreementDocument` better?

Reply:
When is it minimized to "a" its becomes real short :)

It also reflects DB model so name change would require migration for a reduction of 10 characters